src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -1114,6 +1114,18 @@ class TaskStatus(Model):
   time = ndb.DateTimeProperty()
 
 
+class WindowRateLimitTask(Model):
```

Comment:
will these have asome kind of TTL set to auto cleanup?

Reply:
Yes. I've mentioned this in the docstring now.